app/views/layout.erb

```diff
@@ -5,23 +5,24 @@
   <meta charset="utf-8">
   <meta http-equiv="x-ua-compatible" content="ie=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-
-  <title>Simple Photo Processor</title>
-
+  <title>Simple TODO App</title>
   <link rel="stylesheet" href="/stylesheets/main.css">
   <link rel="icon" href="/images/favicon.png'">
 </head>
 
 <body>
   <div class="container">
     <strong>Together, we can TODO it!</strong>
-    <a href="/users/new">Sign Up</a>
-    <a href="/sessions/new">Sign In</a>
+    <% if session[:user_id] %>
```

Comment:
Having a `current_user` helper would be an improvement on looking directly at the session here.